ArmVirtPkg/ArmVirtKvmTool.dsc

```diff
@@ -87,6 +87,12 @@
   ArmMonitorLib|ArmPkg/Library/ArmMonitorLib/ArmMonitorLib.inf
   ArmTrngLib|ArmPkg/Library/ArmTrngLib/ArmTrngLib.inf
 
+[LibraryClasses.common.SEC]

```

Comment:
Would it be possible to add these changes to ArmVirtPkg/ArmVirt.dsc.inc ?
That way we can have all these settings in a common place.